app-modules/alert/src/Policies/AlertPolicy.php

```diff
@@ -73,8 +73,12 @@ public function view(Authenticatable $authenticatable, Alert $alert): Response
         );
     }
 
-    public function create(Authenticatable $authenticatable): Response
+    public function create(Authenticatable $authenticatable, $prospect = null): Response
     {
+        if ($prospect instanceof Prospect && $prospect->student()->exists()) {
+            return Response::deny('You cannot create alert as Prospect has been converted to a Student.');
+        }
```

Comment:
Hi @Orrison ,

For the common relation manager this will not work and it resulted the following error.

https://flareapp.io/share/47qyylW7

`public function create(Authenticatable $authenticatable, ?Prospect $prospect = null): Response`

in that case we need to go with that `if ($prospect instanceof Prospect)` condition ?